sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -1260,8 +1260,12 @@ public partial class BlockBlobClient : Azure.Storage.Blobs.Specialized.BlobBaseC
         public BlockBlobClient(System.Uri blobUri, Azure.Storage.Blobs.BlobClientOptions options = null) { }
         public BlockBlobClient(System.Uri blobUri, Azure.Storage.StorageSharedKeyCredential credential, Azure.Storage.Blobs.BlobClientOptions options = null) { }
         public virtual int BlockBlobMaxBlocks { get { throw null; } }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
         public virtual int BlockBlobMaxStageBlockBytes { get { throw null; } }
+        public virtual long BlockBlobMaxStageBlockBytesLong { get { throw null; } }
```

Comment:
The convention in .NET is `Foo` and `LongFoo` but you've got `FooLong` here.  See [Array.LongLength](https://docs.microsoft.com/en-us/dotnet/api/system.array.longlength?view=netcore-3.1) for comparison.  We're hiding the old `Foo` name so that gives  us a little more flexibility.  Maybe we call it `BlockBlobMaxStageBlockLongBytes` so the convention is still consistent but it'll still be very obvious to customers who need to change?